Jenkinsfile

```diff
@@ -44,22 +44,21 @@ pipeline {
         stage('Deliver Docker images') {
           when {
             anyOf {
-              branch 'master'
+              branch 'postgresql'
               buildingTag()
             }
           }
           steps {
             script {
-              env.DOCKER_TAG = 'branch-master'
+              env.DOCKER_TAG = 'branch-postgresql'
               if (env.TAG_NAME) {
```

Comment:
Let's not have tags for now

Reply:
When this pr merged, the Jenkins will create a new tags and push to dockerhub.

Reply:
That if is if we detect a tag on the linagora repo being built (like the tags for releases for example). 

Let's not have this for now I think, it's just experimental and that's it atm :)